client/scripts/DrawTogether.js

```diff
@@ -1295,6 +1326,262 @@ DrawTogether.prototype.handlePaintSelection = function handlePaintSelection (eve
 	}.bind(this));
 };
 
+// Send null for unchanging value
+DrawTogether.prototype.updateIndividualFavoriteDom = function updateIndividualFavoriteDom(newX, newY, newName, newOwner, element) {
+	if(newX !== null)
+		element.dataset.x = newX;
+	if(newY !== null)
+		element.dataset.y = newY;
+	if(newName !== null)
+		element.dataset.name = newName;
+	if(newOwner !== null)
+		element.dataset.owner = newOwner;
+	
+	var coordinateButton = element.getElementsByClassName("fav-coor-button")[0];
+	var inputRename = element.getElementsByClassName("fav-rename-input")[0];
+	if ( (newName || element.dataset.name) === ""){
+		inputRename.value = "";
+		coordinateButton.innerHTML = (newX || element.dataset.x) + "," + (newY || element.dataset.y);
```

Comment:
Yea but I'm not sure textContent works in the others. Also it's cleaner to use the DOM nodes imo